src/generate.ts

```diff
@@ -60,6 +69,18 @@ export function createGeneratePlugin({
       if (!isolatedDeclaration) {
         initTs()
       }
+
+      if (!inputAlias && isPlainObject(options.input)) {
+        inputAlias = options.input
+      }
+
+      if (inputAlias) {
+        const cwd = options.cwd || process.cwd()
+        for (const [fileName, inputFilePath] of Object.entries(inputAlias)) {
+          const id = path.resolve(cwd, inputFilePath)
```

Comment:
I'm not sure whether it's the correct way to generate rollup/rolldown's `id` string.

Reply:
I would suggest using `this.resolve` in `resolveId` to generate the id.

This API provided by rollup/rolldown can be considered reliable, whereas with `path.resolve(cwd, ...)` there may be some edge cases that cannot be covered.